mssql-plugin/widgets/SqlServer-action.json

```diff
@@ -46,6 +46,15 @@
     {
       "label": "Credentials",
       "properties": [
+        {
+          "label": "Authentication Type",
+          "name": "authentication",
+          "widget-type": "select",
```

Comment:
I think an inline radio-group would be better than select, since there are only 2 options but it doesn't really fit a toggle.

Reply:
Changed.